gwu-libraries/sfm-ui
# T220 hide request export button. closes #220

This disables the "Request Export" button on Seedset detail screen where harvest for the seedset is incomplete, failed or has not started yet.

Steps to Test:
1. Log in to sfm-ui, create credentials, and collection. Add seedset to the collect.
2. Go to seedset details screen and check if the request export button is disabled.
3. Add seeds to the seedset and update seedset to initiate a harvest or initiate a harvest from admin.
4. After a minute when the harvest is completed. Check if the harvest's status is completed successful. If so, then the "Request Export" button should be enabled on Seedset details screen.
